packages/editor-ui/src/api/workflows.ts

```diff
@@ -1,11 +1,6 @@
-import { IExecutionsCurrentSummaryExtended, IRestApiContext } from '@/Interface';
-import {
-	ExecutionFilters,
-	ExecutionOptions,
-	ExecutionStatus,
-	IDataObject,
-	WorkflowExecuteMode,
-} from 'n8n-workflow';
+import type { IExecutionsCurrentSummaryExtended, IRestApiContext } from '@/Interface';
+import type { ExecutionFilters, ExecutionOptions, IDataObject } from 'n8n-workflow';
+import { ExecutionStatus, WorkflowExecuteMode } from 'n8n-workflow';
```

Comment:
These two should also be import type, no? But also, they are not used in this file so could be deleted